lib/iris/fileformats/grib/message.py

```diff
@@ -247,12 +258,16 @@ def __getitem__(self, keys):
                 _data[bitmap.astype(bool)] = data
                 # `np.ma.masked_array` masks where input = 1, the opposite of
                 # the behaviour specified by the GRIB spec.
-                data = np.ma.masked_array(_data, mask=np.logical_not(bitmap))
+                data = ma.masked_array(_data, mask=np.logical_not(bitmap))
             else:
                 msg = 'Shapes of data and bitmap do not match.'
                 raise TranslationError(msg)
 
         data = data.reshape(self.shape)
+
+        if ma.isMaskedArray(data):
+            data = array_masked_to_nans(data)
+
```

Comment:
I haven't added this for the GribDataProxy used by GribWrapper for GRIB edition 1 (see `lib/iris/fileformats/grib/__init__.py`) which I think is probably a mistake, concerninly no tests have failed because of this! Have we not tested deffered loading for grib edition 1?